editoast/src/converters/utils.rs

```diff
@@ -440,21 +439,9 @@ fn sncf_extensions(node: &Node) -> SignalSncfExtension {
         })
         .unwrap_or_default();
     SignalSncfExtension {
-        aspects: vec![
-            "Carré".to_string(),
-            "Feu Rouge Clignotant".to_string(),
-            "Sémaphore".to_string(),
-            "Avertissement".to_string(),
-            "Feu Vert".to_string(),
-        ],
-        default_aspect: "CARRE".to_string(),
-        installation_type: "CARRE".to_string(),
-        is_in_service: true,
-        is_lightable: true,
-        is_operational: true,
         label,
         side,
-        ..Default::default()
+        kp: "0+00".into(),
```

Comment:
Everywhere else we put empty strings when the KP is unknown. Should we change those too?